src/App.tsx

```diff
@@ -15,48 +18,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [isReversed, Reverse] = useState(false);
```

Comment:
Maybe it's not critical, but usually, we use the prefix `set` for the function returned by `useState` to change the state. In the case of `Reverse` in the code below it's not really clear to me what function `Reverse` means, so I need to find it. But as I said before, maybe, it's not critical, and it's only my paranoia

Reply:
thanks, good comment